packages/rum/src/boot/recorderApi.spec.ts

```diff
@@ -19,7 +20,22 @@ describe('makeRecorderApi', () => {
   let recorderApi: RecorderApi
   let startRecordingSpy: jasmine.Spy<StartRecording>
   let stopRecordingSpy: jasmine.Spy<() => void>
-  let getDeflateWorkerSingletonSpy: jasmine.Spy<() => DeflateWorker | undefined>
+  let startDeflateWorkerSpy: jasmine.Spy<typeof startDeflateWorker>
+
+  function startDeflateWorkerWith(worker?: DeflateWorker) {
+    if (!startDeflateWorkerSpy) {
+      startDeflateWorkerSpy = jasmine.createSpy<typeof startDeflateWorker>('startDeflateWorker')
+    }
+    startDeflateWorkerSpy.and.callFake((callback) => callback(worker))
+  }
+
+  function callDeflateCallbackWith(worker?: DeflateWorker) {
```

Comment:
- do we need the worker parameter?
- what about renaming the function to something more domain related? maybe something like `triggerLastRegisteredInitialisationCallback()`